Resources/Prototypes/DeltaV/Roles/Jobs/Fun/misc_startinggear.yml

```diff
@@ -63,4 +63,23 @@
     gloves: ClothingHandsGlovesCombat
     shoes: ClothingShoesSlippers
     id: SyndiPDA
-  innerClothingSkirt: ClothingUniformJumpsuitPyjamaSyndicatePink
\ No newline at end of file
+  innerClothingSkirt: ClothingUniformJumpsuitPyjamaSyndicatePink
+  
+# Syndicate Paramedic
+
+- type: startingGear
+  id: SyndicateParamedicGear
+  equipment:
+    jumpsuit: UniformScrubsColorCybersun
+    back: ClothingBackpackDuffelSyndicateFilledMedical
+    shoes: ClothingShoesChameleonNoSlips
+    ears: ClothingHeadsetAltSyndicate
+    eyes: ClothingEyesHudSyndicateMedical
+    head: ClothingHeadHatSurgcapCybersun
+    outerClothing: ClothingOuterCoatCybersunWindbreaker
+    gloves: ClothingHandsGlovesCombat
+    id: SyndiPDA
+    belt: ClothingBeltParamedicFilled
+    pocket1: HandheldGPSBasic # DeltaV - Give Paramedics useful tools on spawn
```

Comment:
dont need comments since its not crewmate paramedic and is its own gear

Reply:
lol forgot to delete it from copy & paste